Index: Code/src/test/java/XP_Metrics/EvaluateXPTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package XP_Metrics;\r\n\r\nimport org.junit.jupiter.api.AfterEach;\r\nimport org.junit.jupiter.api.BeforeEach;\r\nimport org.junit.jupiter.api.Test;\r\n\r\nimport static org.junit.jupiter.api.Assertions.*;\r\n\r\nclass EvaluateXPTest {\r\n\r\n\r\n    @Test\r\n    void normalisedScore() {\r\n    }\r\n\r\n    @BeforeEach\r\n    void setUp() {\r\n    }\r\n\r\n    @AfterEach\r\n    void tearDown() {\r\n    }\r\n\r\n    @Test\r\n    void testNormalisedScore() {\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Code/src/test/java/XP_Metrics/EvaluateXPTest.java b/Code/src/test/java/XP_Metrics/EvaluateXPTest.java
--- a/Code/src/test/java/XP_Metrics/EvaluateXPTest.java	
+++ b/Code/src/test/java/XP_Metrics/EvaluateXPTest.java	
@@ -4,13 +4,20 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.*;
+import java.util.Arrays;
 
 class EvaluateXPTest {
 
 
     @Test
     void normalisedScore() {
+        int [][] weights = {{1,2,3}, {4,5,6}, {7,8,9}};
+        int weightTotal = Arrays.stream(weights).sum;
+        int result = 0;
+        for (int i=0; i<weights.length; i++){
+            result += (int) (0.25*(weights[i]/weightTotal));
+        }
+        return result;
     }
 
     @BeforeEach
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/Code/.idea/inspectionProfiles/Project_Default.xml\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	
+++ b/.gitignore	
@@ -1,1 +1,2 @@
 /Code/.idea/inspectionProfiles/Project_Default.xml
+/tuleap pull pass.txt
